src/app/locales/zh-tw/settings.yml

```diff
@@ -99,6 +97,23 @@ streaming:
         parameters:
             title: 自訂參數
             placeholder: 加入自訂參數
+    webbrowser:
+        title: Streamlink 網頁瀏覽器 API
```

Comment:
Please keep the term "webbrowser API" untranslated because it's the name of the specific Streamlink interface. It's using the single word "webbrowser" for this very reason.